doc/source/conf.py

```diff
@@ -1,271 +1,64 @@
-# -*- coding: utf-8 -*-
-#
-# flatsurf documentation build configuration file, created by
-# sphinx-quickstart on Thu Mar 17 12:34:49 2016.
-#
-# This file is execfile()d with the current directory set to its
-# containing dir.
-#
-# Note that not all possible configuration values are present in this
-# autogenerated file.
-#
-# All configuration values have a default; values that are commented out
-# serve to show the default.
-
-try:
-    import sage.all
-except ImportError:
-    raise RuntimeError("to build the documentation you need to be inside a Sage shell (run first the command 'sage -sh' in a shell")
-import sys
-import os
+# -- Project information -----------------------------------------------------
 
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
+project = 'surface_dynamics'
```

Comment:
Your sphinx configuration did not work for me.

Since it also talked about flatsurf, it probably was outdated anyway?

Reply:
It used to work for me. It is fine to refresh it.